### ISSUE TITLE
Feature request: Toggle 'automatic updating' per item

### ISSUE THREAD

**Issue by ZoharTelor** [DWilliames/paddy-sketch-plugin#64]
### Feature request / enhancement

Hey! :)

I love that you added "Enable/Disable automatic update" feature it's great!
It would be even better if we could toggle it on and off for each paddy group we create.. because sometimes we need to create buttons or elements which have a minimum size (which is different than the paddings) and we want to resize them manually.

Something similar to how selection in groups work in sketch:
![image](https://user-images.githubusercontent.com/29888529/36966483-994451c0-2065-11e8-8471-5c296e413d4b.png)
We can toggle on or off how selection is made for each particular group instead of a general settings for all groups.

I hope it's something that's possible within sketch's framework!

Thanks,
Zohar

**Comment by williambout** [comment]
Would love to have that too!

**Comment by DWilliames** [comment]
Great idea @ZoharTelor!

It is possible to add something like that to the inspector for Paddy... however, how Paddy is currently setup, it would be quite tricky.

I don't think this feature request would work well unless I could implement it as you have suggested. Otherwise I feel it would be quote cumbersome if you had to select that from the a drop down menu within Paddy.

**Comment by ZoharTelor** [comment]
Yeah, you're right... but using a shortcut to toggle it on or off might not be that bad.
